Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md

```diff
@@ -4,11 +4,21 @@ versionFrom: 8.0.0
 
 # reCAPTCHA V3
 
-You need to configure your site keys adding your public and private keys in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
+You need to configure your site keys by adding your public and private keys in the `appSettings.json` file:
 
-```xml
-<setting key="RecaptchaV3SiteKey" value="..." />
-<setting key="RecaptchaV3PrivateKey" value="..." />
```

Comment:
The same here, unless we make a specific article for V9, we need to still have the configuration for V8 in the article as well :)

Reply:
Done